### PR TITLE
DOCS-11556 add note about vcore allocation warnings in docs

### PR DESCRIPTION
**Pull request by IsaacEldridge**
@mauro-dentoni  this PR just adds indication in docs that they'll get an error warning now

### CODE REVIEW

**Review by mauro-dentoni — APPROVED** [review]

**Comment by mauro-dentoni** [comment]
@IsaacEldridge Having an indication that an error warning will be displayed, when no vCores are assigned, sounds fine to me. Thanks

File: modules/ROOT/pages/add-api-to-unified-schema.adoc [hunk 1]
```diff
@@ -25,7 +25,7 @@ However, it need not be a Mule application or be running on Mule runtime engine
 Additionally you must:
 
 * Have xref:permissions.adoc[the correct permissions] to add APIs.
-* Ensure that the business group or organization using Anypoint DataGraph has the correct number of vCores allocated, as specified in your plan.
+* Ensure that the business group or organization using Anypoint DataGraph has the correct number of vCores allocated, as specified in your plan. If you haven't allocated the correct number of vCores to the environment in which you're attempting to add an API, the UI raises an error message.
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/add-api-to-unified-schema.adoc:28]:
```suggestion
* Ensure that the business group or organization using Anypoint DataGraph has the correct number of vCores allocated, as specified in your plan. If you haven't allocated the correct number of vCores to the environment in which you're attempting to add an API, Anypoint DataGraph raises an error message.
```

File: modules/ROOT/pages/datagraph-qsg.adoc [hunk 2]
```diff
@@ -10,10 +10,12 @@ Getting started with Anypoint DataGraph involves:
 
 == Prerequisites
 
+Before getting started, ensure you have:
+
 * Familiarity with Anypoint Platform and Exchange, including permissions to use Exchange to publish and view RAML or OAS specifications in your business group
 * An Anypoint Platform account
 * Admin or Contribute permissions
-* The correct number of vCores allocated to your business group or organization using Anypoint DataGraph, as specified in your plan.
+* The correct number of vCores allocated to your business group or organization using Anypoint DataGraph, as specified in your plan. If you haven't allocated the correct number of vCores to the environment in which you're attempting to add an API, the UI raises an error message.
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/datagraph-qsg.adoc:18]:
```suggestion
* The correct number of vCores allocated to your business group or organization using Anypoint DataGraph, as specified in your plan. If you haven't allocated the correct number of vCores to the environment in which you're attempting to add an API, Anypoint DataGraph raises an error message.
```

File: modules/ROOT/pages/index.adoc [hunk 3]
```diff
@@ -112,7 +112,7 @@ To use Anypoint DataGraph, ensure you meet the following requirements:
 The API calls associated with your Aypoint DataGraph license refer to calls made by Anypoint DataGraph to downstream REST APIs.
 --
 
-* Your plan must allocate the correct number of vCores for your business group or organization using Anypoint DataGraph.
+* Your plan must allocate the correct number of vCores for your business group or organization using Anypoint DataGraph. If you haven't allocated the correct number of vCores to the environment in which you're attempting to add an API, the UI raises an error message. The UI enables you to reapply the same changes after allocating the correct number of vCores.
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/index.adoc:115]:
```suggestion
* Your plan must allocate the correct number of vCores for your business group or organization using Anypoint DataGraph. If you haven't allocated the correct number of vCores to the environment in which you're attempting to add an API, Anypoint DataGraph raises an error message. Reapply the same changes after allocating the correct number of vCores.
```

**Review by hannanelson — APPROVED** [review]

**Pull request merged**